### PR TITLE
FI-511 Apply bulk_access_token to Group export

### PR DESCRIPTION
**Pull request by yunwwang**
This pull request contains changes for FI-511. Bulk data export sequence can use bulk_access_token if available.

**Submitter:**
- [x] This pull request describes why these changes were made
- [x] Internal ticket for this PR:
https://oncprojectracking.healthit.gov/support/browse/FI-511
- [x] Internal ticket links to this PR
- [x] Internal ticket is properly labeled (Community/Program)
- [x] Internal ticket has a justification for its Community/Program label
- [x] Code diff has been reviewed for extraneous/missing code
- [x] Tests are included and test edge cases
- [x] Tests/code quality metrics have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/modules/bulk_data/bulk_data_export_sequence.rb [hunk 1]
```diff
@@ -154,10 +159,10 @@ def check_cancel_request
         end
 
         @client.set_no_auth
-        skip 'Could not verify this functionality when bearer token is not set' if @instance.token.blank?
+        skip 'Could not verify this functionality when bearer token is not set' if @instance.bulk_access_token.blank?
```
Comment (Jammjammjamm) [inline lib/modules/bulk_data/bulk_data_export_sequence.rb]: Don't most of the bulk data tests need a skip like this?

Comment (yunwwang) [reply lib/modules/bulk_data/bulk_data_export_sequence.rb]: bulk data allows open access without access token
